backend/database/versions/34_2be441104b48_standardize_db_fields.py

```diff
@@ -0,0 +1,60 @@
+"""standardize db fields
+
+Revision ID: 34_2be441104b48
+Revises: 33_c5aaf6e2ca9e
+Create Date: 2023-01-09 16:40:13.184800
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '34_2be441104b48'
+down_revision = '33_c5aaf6e2ca9e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('Collection', sa.Column('tombstone', sa.BOOLEAN(), nullable=True), schema='persistence_schema')
+    op.drop_column('Collection', 'tombstoned', schema='persistence_schema')
```

Comment:
Where are we copying the old column to the new column? I think running this migration will result in data loss. You need to [alter_column](https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.BatchOperations.alter_column)